openmetadata-ui/src/main/resources/ui/src/components/DataQuality/SummaryPannel/SummaryPanel.component.tsx

```diff
@@ -40,8 +43,10 @@ export const SummaryPanel = ({ testSuiteId }: SummaryPanelProps) => {
   };
 
   useEffect(() => {
-    fetchTestSummary();
-  }, [testSuiteId]);
+    if (testCasePermission?.ViewAll || testCasePermission?.ViewBasic) {
```

Comment:
Isn't it should be `ViewTests`?
<img width="383" alt="Screenshot 2023-06-26 at 9 39 03 PM" src="https://github.com/open-metadata/OpenMetadata/assets/12962843/e3a9abbc-cfcd-4be0-97cd-292eb3f5a8fd">

Reply:
Same here checking from resource permission